### PR TITLE
699 alert on widget delete

### PR DESCRIPTION
**Pull request by komalahuwalia**
Before your pull request is reviewed and merged, please ensure that:

* [ ] there are no linting errors
* [ ] your code is in a uniquely-named feature branch and has been rebased on top of the latest master. If you're asked to make more changes, make sure you rebase onto master then too!
* [ ] your pull request is descriptively named and links to an issue number, i.e. `Fixes #123`
* [ ] **Make sure you have run the tests!!**

Thank you!

### CODE REVIEW

File: client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    this.props.closeModal();
+  }
+
+  render() {
+    return (
+      <div className="deleteWidgetWarning-modal__content">
+        <h1 className="deleteWidgetWarning-modal__sub-title">
+          <div>
+            <div className="deleteWidgetWarning-modal__sub-title">
+        Are you sure you want to delete the widget?
```
Comment (MathuraMG) [inline client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx]: Let's maybe rephrase it to say `component` instead of `widget` ? This is just aa thought, but we can also wait for Esther and Regina to have a look before making this change

Comment (komalahuwalia) [reply client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx]: Lets me know what shall the final message be

File: client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            </div>
+            <div className="deleteWidgetWarning-modal__footnote">
+        This would delete the widget from your page permanently
+            </div>
+          </div>
+        </h1>
+        <button
```
[the remark this answers is not in the document]
Comment (komalahuwalia) [reply client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx]: Done

File: client/src/app/components/App/Modal/DeleteWidgetWarning/deleteWidgetWarning.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    font-weight: 400;
+    font-size: $small-font-size;
+  }
+
+  &__label{
+    @extend %modal__label;
+  }
+
+  &__button{
+    @extend %modal__button;
```
[the remark this answers is not in the document]
Comment (komalahuwalia) [reply client/src/app/components/App/Modal/DeleteWidgetWarning/deleteWidgetWarning.scss]: Done

File: client/src/app/reducers/editors.js [hunk 4]
```diff
@@ -10,7 +10,9 @@ import {
 const initialState = {
   editors: {},
   editorIndex: 0,
-  currentWidget: ''
+  currentWidget: '',
+  deleteWidgetWarning: false,
```
[the remark this answers is not in the document]
Comment (komalahuwalia) [reply client/src/app/reducers/editors.js]: Done

File: client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+export default class DeleteWidgetWarning extends React.Component {
+  deleteWidgetConfirm = () => {
+    this.props.deleteWidget();
+    this.props.closeModal();
+  }
+
+  render() {
+    return (
+      <div className="deleteWidgetWarning-modal__content">
```
[the remark this answers is not in the document]
Comment (komalahuwalia) [reply client/src/app/components/App/Modal/DeleteWidgetWarning/DeleteWidgetWarning.jsx]: Done

**Comment by MathuraMG** [comment]
fixes #699
